pom.xml

```diff
@@ -26,8 +26,8 @@
   <properties>
     <changelist>999999-SNAPSHOT</changelist>
     <gitHubRepo>jenkinsci/plugin-compat-tester</gitHubRepo>
-    <logbackVersion>1.4.5</logbackVersion>
     <maven.scm.providers.version>1.6</maven.scm.providers.version>
+    <slf4j.version>2.0.6</slf4j.version>
```

Comment:
Since we now need to define this version for both `slf4j-api` and `slf4j-jdk14`, extracting it to a property to eliminate duplication. This matches what we do in Jenkins core.